package.json

```diff
@@ -39,15 +39,15 @@
     "sprucebot-node": "^0.4.2",
     "sprucebot-skills-kit-server": "^1.0.1",
     "styled-components": "^2.2.2",
-    "universal-cookie": "^2.1.0",
-    "enzyme": "^3.3.0",
-    "enzyme-adapter-react-16": "^1.1.1",
-    "enzyme-to-json": "^3.3.1"
+    "universal-cookie": "^2.1.0"
   },
   "devDependencies": {
     "babel-jest": "^22.0.4",
     "babel-preset-es2015": "^6.24.1",
     "babel-preset-react": "^6.24.1",
+    "enzyme": "^3.3.0",
```

Comment:
Having these as devDependencies has caused deployments in non-development environments to fail.   I'm not sure why enzyme is kicking in at all but this is a show stopper type problem so the underlying issue needs to be resolved before we can do this.

Reply:
Can you point us to some logs or show some screens of the failures?

Reply:
on awards *this is from a while back) -        $ npm run build
      
       > sprucebot-skills-kit@0.5.0 build /tmp/build_e4e23086d12e4f7f231857a38c68ea2d/sprucelabsai-sprucebot-skill-awards-2443fd3ffdefd2dc6e2912235939cec2c98b5179
       > next build interface
      
       > Using external babel configuration
       > Location: “/tmp/build_e4e23086d12e4f7f231857a38c68ea2d/sprucelabsai-sprucebot-skill-awards-2443fd3ffdefd2dc6e2912235939cec2c98b5179/.babelrc”
       > Using “webpack” config function defined in next.config.js.
       > Failed to build on /tmp/caf2f9b8-442c-4232-bf9e-69b1e7f93e4d
       { Error: ./interface/pages/index.test.js?entry
       Module not found: Error: Can’t resolve ‘enzyme’ in ‘/tmp/build_e4e23086d12e4f7f231857a38c68ea2d/sprucelabsai-sprucebot-skill-awards-2443fd3ffdefd2dc6e2912235939cec2c98b5179/interface/pages’

Reply:
Looks like we need to tell next.js to ignore glob `pages/**.test.js`